packages/gatsby-theme-newrelic/src/components/DarkModeToggle.js

```diff
@@ -56,11 +56,37 @@ const DarkModeToggle = ({ className, size, onClick }) => {
       `}
     >
       <Icon
-        name={darkMode.value ? 'nr-dark-mode' : 'nr-light-mode'}
+        name="nr-light-mode"
```

Comment:
should we change the name of this icon to something like `nr-dark-mode-toggle` or vice versa?

Reply:
we should, its not defined in this repo though i don't think? at least I can't find it

Reply:
lol it is, i fixed